ci-operator/templates/cluster-launch-e2e-upgrade.yaml

```diff
@@ -192,9 +192,9 @@ objects:
         trap 'rc=$?; if [[ $rc -ne 0 ]]; then touch /tmp/shared/exit; else touch /tmp/shared/install; fi; exit $rc' EXIT
         trap 'kill $(jobs -p); exit 0' TERM
 
-        oreg_url="${IMAGE_FORMAT}"
+        oreg_url='${IMAGE_FORMAT}'
         if [[ -n "${PREVIOUS_IMAGE_TAG}" ]]; then
-          oreg_url="'docker.io/openshift/origin-${component}:'${PREVIOUS_IMAGE_TAG}"
+          oreg_url='docker.io/openshift/origin-${component}:'"${PREVIOUS_IMAGE_TAG}"
```

Comment:
Let's use single quotes around `${PREVIOUS_IMAGE_TAG}` on lines 196-197 since it's a template (not bash) substitution:

```sh
if [[ -n '${PREVIOUS_IMAGE_TAG}' ]]; then
  oreg_url='docker.io/openshift/origin-${component}:${PREVIOUS_IMAGE_TAG}'

Reply:
Good idea, implemented that